### PR TITLE
{bio}[foss/2019a] phyx v1.01

### PR DESCRIPTION
**Pull request by omula**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/phyx/phyx-1.01-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+dependencies = [
+    ('Armadillo', '9.700.2'),
+    ('NLopt', '2.6.1'),
+]
+
+start_dir = 'src'
+
+preinstallopts = 'mkdir %(installdir)s/bin && '
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/p/phyx/phyx-1.01-foss-2019a.eb]: @omula Please remove duplicate empty line

File: easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+diff -Naur phyx-1.01.orig/src/Makefile.in phyx-1.01/src/Makefile.in
```
Comment (boegel) [inline easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch]: @omula Please add missing patch description + author or upstream reference.

Also, shouldn't this be `-lopenblas` instead? Do you really need to link to ScaLAPACK?

Comment (omula) [reply easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch]: Thanks @boegel 
You are right. It links with openblas I have checked with ldd.

Comment (boegel) [inline easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch]: @omula This patch should be removed, no longer used?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4ede7bcc34f563c928ef694fdba83625 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2727.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/714a621bddc0855052f82099bd84b1c5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22695e0f2a766b903ac63c7963ef7310 for a full test report.

File: easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch (hunk 2 above)
Comment (omula) [reply easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch]: @boegel you are right. done

**Comment by boegel** [comment]
Going in, thanks @omula!

**Pull request merged**